src/azul/azulclient.py

```diff
@@ -69,41 +74,57 @@ def _make_notification(self, bundle_fqid):
                 "bundle_uuid": bundle_uuid,
                 "bundle_version": bundle_version
             },
-            "test_name": self.test_name
         }
 
     def reindex(self, sync: bool = False):
-        errors = defaultdict(int)
-        missing = {}
-        indexed = 0
-        total = 0
-
         logger.info('Querying DSS using %s', json.dumps(self.query(), indent=4))
         bundle_fqids = self._post_dss_search()
         logger.info("Bundle FQIDs to index: %i", len(bundle_fqids))
+        notifications = [self._make_notification(fqid) for fqid in bundle_fqids]
+        self._reindex(notifications, sync=sync)
+
+    def test_reindex(self, test_name: str, test_uuid: str, sync: bool = False) -> Set[str]:
+        logger.info('Querying DSS using %s', json.dumps(self.query(), indent=4))
+        real_bundle_fqids = self._post_dss_search()
+        logger.info("Bundle FQIDs to index: %i", len(real_bundle_fqids))
+        notifications = []
+        effective_bundle_fqids = set()
+        for bundle_fqid in real_bundle_fqids:
+            new_bundle_uuid = str(uuid.uuid4())
+            _, _, version = bundle_fqid.partition('.')
+            effective_bundle_fqids.add(new_bundle_uuid + '.' + version)
+            notifications.append(self._make_test_notification(bundle_fqid, new_bundle_uuid, test_name, test_uuid))
```

Comment:
Inlining the method would brake this long, harder to read line into multiple lines.

Reply:
Done. Sorry I missed this review comment before.